assets/js/components/settings/settings-module.js

```diff
@@ -340,7 +336,6 @@ class SettingsModule extends Component {
 									{
 										hasSettings && ! setupComplete &&
 											<ModuleSetupIncomplete
-												screenID={ screenID }
```

Comment:
The only reason this was still being passed is because the old `AdSenseSettingsStatus` component used it. Now that that is gone, we can eliminate this prop.